<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditTypeDevice</class>
 <widget class="QDialog" name="AddEditTypeDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование типов устройств</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/22x22/hardware/ico/hardware_22x22.png</normaloff>:/22x22/hardware/ico/hardware_22x22.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableView" name="typeDeviceView">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/add/ico/edit_add_16x16.png</normaloff>:/16x16/add/ico/edit_add_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/del_16x16.png</normaloff>:/16x16/cancelDel/ico/del_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Применить</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/apply/ico/apply_16x16.png</normaloff>:/16x16/apply/ico/apply_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="revertButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/revert/ico/revert_16x16.png</normaloff>:/16x16/revert/ico/revert_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEditIcon">
       <property name="text">
        <string>Изменить
иконку</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/edit/ico/edit_16x16.png</normaloff>:/16x16/edit/ico/edit_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delIcoButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Удалить
иконку</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/del_16x16.png</normaloff>:/16x16/cancelDel/ico/del_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>128</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>AddEditTypeDevice</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
